e2e/terraform/terraform.tfvars

```diff
@@ -0,0 +1,5 @@
+region            = "us-east-1"
+instance_type     = "t2.medium"
+server_count      = "3"
+client_count      = "4"
+nomad_sha         = "c784d59ab3ff1ee929465e499f9c46a14e71bce6"
```

Comment:
CI should dynamically populate this based on an environment variable or Terraform var flag in the apply step. It can reference the current HEAD of the Nomad repo. We should remove it from this file so locally, non-CI, it prompts the user.